melange/init

```diff
@@ -0,0 +1,76 @@
+#!/bin/busybox sh
+set -e
+set -x
+
+/bin/mount -t proc proc -o nodev,nosuid,hidepid=2 /proc
+/bin/mount -t devtmpfs -o nosuid,noexec devtmpfs /dev
```

Comment:
just set `PATH=/usr/bin:/usr/sbin:/sbin:/bin`
and export that and don't call things by their full path.

Reply:
Yea, it's easier, implemented, thanks!